inference-engine/src/gna_plugin/backend/am_intel_dnn.cpp

```diff
@@ -1583,6 +1583,7 @@ void GNAPluginNS::backend::AMIntelDNN::InitGNAStruct(intel_nnet_type_t *ptr_nnet
             case kDnnPiecewiselinearOp:
 #if  GNA_LIB_VER == 2
                 {
+                    IE_ASSERT(gnaOperation->Operands != nullptr);
                     auto& outputTensor = const_cast<Gna2Tensor&>(*gnaOperation->Operands[OutOpIdx]);
```

Comment:
Should we also check size of Operands to make sure that it has value with OutOpIdx inside of it?

Reply:
Let's do this in the PR for master only - KW should be OK with this check

Reply:
This comment is addressed.